.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+<!-- Please describe your pull request here. -->
+
+## References
+
+- TODO
+
+<!-- References to relevant GitHub issues and pull requests, esp. upstream and downstream changes -->
+
+## Submitter checklist
+
+- [ ] The PR request is well described, including the body and the references
```

Comment:
Maybe we could explicitly ask for a justification/use case description, either directly or via linked issue?
Too often we get PRs where it's not at all obvious what problem is being solved.

Reply:
```suggestion
- [ ] The PR request is well described and justified, including the body and the references
```